src/utils.js

```diff
@@ -432,6 +432,12 @@
             });
             return dat;
           },
+
+          /**
+           * AngularFire version number.
```

Comment:
Just as a general thought on Catapult; I think we should probably note something to the affect that "This is automatically set by build tools during release." so we don't get so many support-related inquiries about why the version numbers are incorrect.